docs/GettingStarted.md

```diff
@@ -0,0 +1,26 @@
+﻿# Getting Started
+
+The [aspnet/SignalR](https://github.com/aspnet/SignalR) repository is part of the larger ecosystem of [aspnet](https://github.com/aspnet) repositories. As such, it shares a somewhat unique build process that has a few dependencies you must have installed and a couple of techniques you should familiarize yourself with.
+
+## Dependencies
+In order to successfully build and test *SignalR* you'll need to ensure that your system has the following installed:
+* Docker
+* [NodeJS](https://nodejs.org/) version 6.9 or later
+* NPM *(typically bundled with NodeJS)*
+
+## How To Build
+Full instructions for how to build repositories within the [aspnet](https://github.com/aspnet) family of repositories can be found [in  the Home repository](https://github.com/aspnet/Home/wiki/Building-from-source).
+
+The short-hand version of that is to simply run the  `build.cmd` or `build.sh` script that's in the root of the *SignalR* repository.
+
+The build process is handled by *KoreBuild* which can be found in the [aspnet/BuildTools](https://github.com/aspnet/BuildTools) repository, documentation for which can be found [here](https://github.com/aspnet/BuildTools/tree/dev/docs).
```

Comment:
I don't think it's necessary to put in detailed information about KoreBuild. That's purely an internal implementation detail.